sound/soc/sof/sof-pci-dev.c

```diff
@@ -23,6 +23,10 @@ static char *fw_path;
 module_param(fw_path, charp, 0444);
```

Comment:
minor nitpick: I didn't understand "The firmware path can be used" actually refers to the "fw_path" module variable, so the rest of commit message didn't make sense at first. Using a firmware path to load firmware files seems kind of obvious until you realize it's a mechanism to use a different path. But really optional, people will understand after they look at the code.

Reply:
reworded.